packages/server/src/connection-server.ts

```diff
@@ -35,11 +35,14 @@ export function* createConnectionServer(options: ConnectionServerOptions): Opera
         }
       });
 
-      yield subscribe(connection.events).forEach(function*(message: TestEvent) {
+      let close: CloseEvent = yield subscribe(connection.events).forEach(function*(message: TestEvent) {
```

Comment:
Why not just `let { code, reason }: CloseEvent = `?

Reply:
Excellent point!